python/cloud-run-python-hello-world/cloudbuild.yaml

```diff
@@ -0,0 +1,30 @@
+steps:
+- id: docker build
+  name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '-t', 'gcr.io/${_PROJECT_ID}/${_SERVICE}', '.']
+
+- id: docker push
+  name: 'gcr.io/cloud-builders/docker'
+  args: ['push', 'gcr.io/${_PROJECT_ID}/${_SERVICE}']
+
+- id: Deploy to Cloud Run
+  name: 'gcr.io/cloud-builders/gcloud'
+  args:
+  - beta
+  - run
+  - deploy
+  - ${_SERVICE}
```

Comment:
This might be an issue in case multiple CI pipelines will run in parallel - they might collision.
We need a way to have each CI run create a separate service that can be tested and deleted without impacting other CI runs.
Look at the similar discussion we had with Adam in Go PR.

Reply:
Done in next commit.